cibuildwheel/__main__.py

```diff
@@ -152,6 +153,7 @@ def main() -> None:
     build_verbosity_str = get_option_from_environment('CIBW_BUILD_VERBOSITY', platform=platform, default='')
 
     build_selector = BuildSelector(build_config, skip_config)
```

Comment:
We should probably rename this `BuildSelector` class, now it's being used for build and test. Could be `IdentifierSelector`?

Reply:
I'd rather wait on this, since I'm thinking that if we have add selectors like `*_universal2/arm64`, we might end up splitting this off into a TestSelector, leaving the BuildSelector alone. Depends on the implementation; if if it easy we could leave it one class and then rename it. Or we can make a `IdentifierSelector` with `BuildSelector` and `TestSelector` subclasses to be ready for potential changes like that. :) (But you seem to not like inheritance?)

Reply:
I've updated this, see if you like it better.  I have an `IdentifierSelector` with `BuildSelector` and `TestSelector`; after Universal2 does in, TestSelector can have extra logic in it if needed for Universal2.

Reply:
I doubt we'd need extra logic, but the different constructor makes some sense.